src/styles/add/card.scss

```diff
@@ -0,0 +1,42 @@
+.cards {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: $card-width;
+  gap: 46px 48px;
+  margin: 50px 40px;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, $card-width);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, $card-width);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, $card-width);
+  }
+}
+
+.button {
```

Comment:
each filename should match BEM block class and contain only 1 block styles

So in this case, the filename should be `cards.scss`, and button written in a separate file (or make button an element of `product` and write button styles there)